app/models/concerns/model/subtransactable.rb

```diff
@@ -15,6 +15,6 @@ module Model::Subtransactable
 		has_one :supporter, through: :trx
 		has_one :nonprofit, through: :trx
 
-		has_many :subtransaction_payments, through: :subtransaction
+		has_many :payments, through: :subtransaction
```

Comment:
I'll be honest, I'm surprised this passes the tests. I had thought you needed to add the class name here like you did in `Subtransaction`.

Reply:
Solved!